_data/simple-icons.json

```diff
@@ -7872,12 +7873,13 @@
         {
             "title": "Samsung",
             "hex": "1428A0",
-            "source": "https://www.samsung.com/us"
+            "source": "https://www.samsung.com/us/about-us/brand-identity/logo/"
```

Comment:
Does it make sense to add this as guidelines as well?

```suggestion
            "source": "https://www.samsung.com/us/about-us/brand-identity/logo/",
            "guidelines": "https://www.samsung.com/us/about-us/brand-identity/logo/"
```

Reply:
Sure, thank you! Dont know how I missed this one!